controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/JobList.java

```diff
@@ -141,24 +141,9 @@ public JobList present() {
             return mapToList(Function.identity());
         }
 
-        /** Returns the subset of jobs where the run of the given type occurred before the given instant */
-        public JobList endedBefore(Instant threshold) {
-            return matching(run -> run.end().orElse(Instant.MAX).isBefore(threshold));
-        }
-
-        /** Returns the subset of jobs where the run of the given type occurred after the given instant */
-        public JobList endedAfter(Instant threshold) {
-            return matching(run -> run.end().orElse(Instant.MIN).isAfter(threshold));
-        }
-
-        /** Returns the subset of jobs where the run of the given type occurred before the given instant */
-        public JobList startedBefore(Instant threshold) {
-            return matching(run -> run.start().isBefore(threshold));
-        }
-
-        /** Returns the subset of jobs where the run of the given type occurred after the given instant */
-        public JobList startedAfter(Instant threshold) {
-            return matching(run -> run.start().isAfter(threshold));
+        /** Returns the subset of jobs where the run of the given type occurred at or before the given instant */
+        public JobList endedNoLaterThan(Instant threshold) {
```

Comment:
Consider naming `endedAtOrBefore` to match doc.

Nit: Javadoc says "of the given type", but there is no type given here that I can tell.

Reply:
Makes sense :)